src/internal/hooks/use-focus-visible-state/index.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { useEffect, useState } from 'react';
+
+import { isModifierKey } from '@cloudscape-design/component-toolkit/internal';
+
+export function useFocusVisibleState() {
```

Comment:
I think we should add unit tests for this hook as it's now a standalone hook outside of the internal chart-plot component.

Reply:
We are not using this anymore, removed the hook